### PR TITLE
Generalized Chart Example Gallery

### PR DESCRIPTION
**Pull request by Matt-Blanco**
Added two prototype examples into the example gallery of the _Rich Screen Reader Experiences for Accessible Data Visualization_ paper. The examples use the generalized accessibility library to create the navigable tree structure rather than the original hard coded prototypes.

### CODE REVIEW

**Comment by Matt-Blanco** [comment]
Accessibility Library Demo:

https://user-images.githubusercontent.com/21320778/168115010-5ed358af-e362-42d7-8896-82f6c3b0a73e.mp4

**Review by arvind — CHANGES_REQUESTED** [review]
Hi @Matt-Blanco, this looks like a great start! A few of suggestions:

1. I think this addition will make most sense at the _end_ of Section 4 rather than where it currently is at the start. 
2. Can we add a button or some other way to set the focus to the prototype? Hitting "tab" to focus on the prototype is pretty tedious. 
3. In your video, it looks like there's a line chart example as well, but I see it commented out in the most recent commit? Did we want to drop that and only show one interactive prototype? Or is there a way to add a dropdown so people can switch between examples? 
4. Are we able to get any of the other examples working? 

(We may have talked about these last two points on Monday and I'm blanking on it, but I wanted to surface it here just in case. Since the MIT News article is scheduled to be published on Monday, I think that's a good deadline for us to finalize this PR by!).

File: Gemfile [hunk 1]
```diff
@@ -29,4 +29,5 @@ end
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
 gem "nokogumbo"
+gem "webrick"
```
Comment (arvind) [inline Gemfile:32]: @Matt-Blanco: I'm trying to figure out what this gem is and how it relates to this PR, could you help me out? Afaict, it starts an HTTP Server?

Comment (Matt-Blanco) [reply Gemfile:32]: I'm not too sure where the discrepancy between gemfiles arose from I just did `gem install` once I installed Ruby and began working on the addition to the paper. The changes to the Gemfile and Gemfile.lock can probably be omitted

Comment (alundgard) [reply Gemfile:32]: I've needed to add the webrick gem as a bug fix for Ruby 3.0. See [here](https://github.com/jekyll/jekyll/issues/8523).

Comment (arvind) [reply Gemfile:32]: Ah good to know, thanks @alundgard!

**Pull request merged**